tests/util.py

```diff
@@ -237,7 +237,7 @@ def get_flags_parser() -> argparse.ArgumentParser:
         "--target-version",
         action="append",
         type=lambda val: TargetVersion[val.upper()],
-        default=(),
+        default=[],
```

Comment:
Seems unnecessary?

Reply:
this was causing some mypy issue locally. I'll check once again.